app/assets/stylesheets/pages/login.scss

```diff
@@ -0,0 +1,32 @@
+///login page styleing
+
+#login {
+  background-image: url("BackgroundImage.jpeg");
+  background-size: cover;
+    .card {
+      border-radius: 12px;
+      margin-top: 30vw;
+      margin-left: 5%;
+      margin-right: 5%;
+      .fa-heart-o{
+        color: white;
+      }
+      .facebook-square{
+        margin: 0;
+      }
+      .btn-warning{
+        margin-top: 2%;
+        margin-bottom: 2%;
+      }
+      .btn-primary{
+        overflow: auto;
+        padding-top: 10px !important;
+        padding-bottom: 10px !important;
```

Comment:
Do we need the `!important` tags? Usually you'll want to avoid these unless absolutely necessary, they're a code smell that you need to add an extra class or make the selector more specific.